templates/default/rt/rtprintindex.html

```diff
@@ -9,7 +9,7 @@ <H1>{$layout.pagetitle}</H1>
     <THEAD>
 	<TR>
 		<TD width="1%">
-			<IMG src="img/ticket.gif" alt="">
+			{icon name="helpdesk"}
```

Comment:
A tu gdzie "fw=true".
Zaczynam dochodzić do wniosku, że chyba, fw trzeba będzie domyślnie uczynić true i tylko czasami dawac fw=false?

Reply:
Właśnie miałem o to zapytać.... bo ja do tej pory patrzyłem wizualnie czy jest taka potrzeba.

Reply:
To i przy okazji {box_row przydałoby się pewnie też dodać icon_class="fa-fw" domyślnie.

Reply:
Ja w paru miejsce dzięki fw=true dostrzegłem różnicę na plus - w formularzach gdzie jest ikonka i etykieta tekstowa.
Być może to zatrze się, gdy przejdziemy na elementy formularzy w których ikonka z etykietą będzie nad kontrolką.

Reply:
> A tu gdzie "fw=true".
> Zaczynam dochodzić do wniosku, że chyba, fw trzeba będzie domyślnie uczynić true i tylko czasami dawac fw=false?

https://github.com/lmsgit/lms/pull/2017/commits/a1b6a0e51b53a382ae505a94c0bad244b682d928